### PR TITLE
add ExecStop directive to cloudgatewaymount systemd service.

### PR DESCRIPTION
**Pull request by gdelafond**
It will unmount the FS and not kill it.

### CODE REVIEW

File: src/scripts/debian/cloudgatewaymount@.service.tmpl [hunk 1]
```diff
@@ -7,6 +7,7 @@ Requires=cloudgateway.service
 #Type=forking
 User=cloudgw
 ExecStart=@CMAKE_INSTALL_PREFIX@/bin/CloudGatewayMount %i @CMAKE_INSTALL_PREFIX@//etc/CloudGatewayConfiguration.xml
+ExecStop=@CMAKE_INSTALL_PREFIX@/bin/CloudGatewayUnmount @CMAKE_INSTALL_PREFIX@//etc/CloudGatewayConfiguration.xml %i
```
Comment (rgacogne) [inline src/scripts/debian/cloudgatewaymount@.service.tmpl:10]: Double `/` here (and above to, but this one is not your doing :)).

**Review by rgacogne — APPROVED** [review]
One non-blocking nit, otherwise looks good.

**Comment by rgacogne** [comment]
Note that looking at the code, we do catch `SIGTERM` so we should properly unmount the filesystem with the current unit file. I would be interested in getting a `strace` of the `FUSE` process being stopped by the current unit file.

**Comment by gdelafond** [comment]
I could not reproduce the problem when I `strace` the process.

Here is what I get when I do not `strace` the process:
```
# df -h | grep Cloud; mount | grep Cloud
# systemctl start cloudgatewaymount@TEST
# df -h | grep Cloud; mount | grep Cloud
CloudGateway:TEST  8.0E     0  8.0E   0% /home/test/public
CloudGateway:TEST on /home/test/public type fuse (rw,nosuid,nodev,relatime,user_id=1001,group_id=1001,default_permissions,allow_other)
# systemctl stop cloudgatewaymount@TEST
# df -h | grep Cloud; mount | grep Cloud
df: /home/test/public: Transport endpoint is not connected
CloudGateway:TEST on /home/test/public type fuse (rw,nosuid,nodev,relatime,user_id=1001,group_id=1001,default_permissions,allow_other)
# 
```

**Comment by rgacogne** [comment]
> Here is what I get when I do not strace the process:

It sure does look like an unclean shutdown. Anything in your logs?

**Comment by gdelafond** [comment]
I did not found anything in the logs (system or CG).

**Comment by gdelafond** [comment]
@rgacogne ok to merge this one?

**Comment by rgacogne** [comment]
I still don't think this should be needed, but I don't have the time to investigate for now so let's merge it. We can revert this change later.

**Pull request merged**